src/server/server.js

```diff
@@ -6,7 +6,19 @@ var app = express();
 var http = require('http').Server(app);
 var io = require('socket.io')(http);
 var SAT = require('sat');
-var sql = require ("mysql");
+var pg = require ('pg');
+var localStorage = require ('node-localstorage');
+
+//Establish database connection
+pg.defaults.ssl = true;
+var client = new pg.Client(process.env.DATABASE_URL);
+client.connect (function (err){
```

Comment:
This still warns users even if they don't want to use a database.

Maybe set the default `config.sqlinfo.host` to an empty string, and only attempt to connect if the config host is not nothing